app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```

Comment:
Do you need this?  It's empty at the moment, so I'd say delete it until you need it (YAGNI)